### PR TITLE
Add minor help documentation changes

### PR DESCRIPTION
**Pull request by skorokithakis**

### CODE REVIEW

File: man_pages/dji_fpv_video_tool-generate-overlay-frames.1 [hunk 1]
```diff
@@ -2,11 +2,11 @@
 .el .ds Aq '
 .TH generate-overlay-frames 1  "generate-overlay-frames " 
 .SH NAME
-generate\-overlay\-frames \- Generates OSD overlay frames
+generate\-overlay\-frames \- Generates single OSD overlay frames
```
Comment (shellixyz) [inline man_pages/dji_fpv_video_tool-generate-overlay-frames.1]: Why single ?

File: man_pages/dji_fpv_video_tool-play-video-with-osd.1 [hunk 2]
```diff
@@ -2,11 +2,11 @@
 .el .ds Aq '
 .TH play-video-with-osd 1  "play-video-with-osd " 
 .SH NAME
-play\-video\-with\-osd \- Plays a video using the MPV video player with OSD by overlaying transparent OSD video in real time
+play\-video\-with\-osd \- Play a DVR video using the MPV player, overlaying the transparent OSD video in real time
```
Comment (shellixyz) [inline man_pages/dji_fpv_video_tool-play-video-with-osd.1]: It doesn't have to be a DVR / video from DJI FPV though, it just need to have a compatible resolution

**Review by shellixyz — COMMENTED** [review]
I only commented on a couple instances which repeat

File: man_pages/dji_fpv_video_tool-generate-overlay-frames.1 (hunk 1 above)
Comment (skorokithakis) [reply man_pages/dji_fpv_video_tool-generate-overlay-frames.1]: As in, not a video, but single frames (images).

File: man_pages/dji_fpv_video_tool-play-video-with-osd.1 (hunk 2 above)
Comment (skorokithakis) [reply man_pages/dji_fpv_video_tool-play-video-with-osd.1]: I wanted to distinguish "raw video from the air unit" from "video generated by this tool".

Comment (skorokithakis) [reply man_pages/dji_fpv_video_tool-play-video-with-osd.1]: I agree the name isn't great, do you have a better idea?

Comment (shellixyz) [reply man_pages/dji_fpv_video_tool-play-video-with-osd.1]: In this case I think the previous version was fine. Just replacing "Plays" with "Play"

"Play a video using the MPV video player with OSD by overlaying a transparent OSD video in real time"

File: man_pages/dji_fpv_video_tool-generate-overlay-frames.1 (hunk 1 above)
Comment (shellixyz) [reply man_pages/dji_fpv_video_tool-generate-overlay-frames.1]: What about this ?

"Generate individual transparent overlay frames as PNG files from a .osd file"

File: man_pages/dji_fpv_video_tool-play-video-with-osd.1 (hunk 2 above)
Comment (skorokithakis) [reply man_pages/dji_fpv_video_tool-play-video-with-osd.1]: 👍

File: man_pages/dji_fpv_video_tool-generate-overlay-frames.1 (hunk 1 above)
Comment (skorokithakis) [reply man_pages/dji_fpv_video_tool-generate-overlay-frames.1]: Ah yes, even better.

File: man_pages/dji_fpv_video_tool-play-video-with-osd.1 (hunk 2 above)
Comment (shellixyz) [reply man_pages/dji_fpv_video_tool-play-video-with-osd.1]: Not sure we need to mention MPV. At least not in the command summary

"Play a video using with OSD by overlaying a transparent OSD video in real time"

Comment (skorokithakis) [reply man_pages/dji_fpv_video_tool-play-video-with-osd.1]: I agree, but was there already so I didn't remove it.

**Comment by shellixyz** [comment]
Merging, thanks @skorokithakis

**Pull request merged**